src/Components/WebAssembly/Sdk/src/GenerateBlazorWebAssemblyBootJson.cs

```diff
@@ -149,7 +149,7 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
 
         private bool IsLazyLoadedAssembly(string fileName)
         {
-            return LazyLoadedAssemblies != null && LazyLoadedAssemblies.Any(a => a.ItemSpec == fileName);
+            return LazyLoadedAssemblies != null && LazyLoadedAssemblies.Any(a => Path.GetFileNameWithoutExtension(a.ItemSpec) == fileName);
```

Comment:
I think we want to either require the extension to be specified or not to be specified at all, I'm not sure if we want to allow both options (with or without the file extension)

Reply:
I guess you could change line 91 to this.

```diff
- if (IsLazyLoadedAssembly(fileName))
+ if (IsLazyLoadedAssembly(resourceName))
```